sunpy/map/mapbase.py

```diff
@@ -518,6 +518,32 @@ def _new_instance(cls, data, meta, plot_settings=None, **kwargs):
             new_map.plot_settings.update(plot_settings)
         return new_map
 
+    @classmethod
+    def notes_fix(cls, notes_doc, datadoc):
+            existing_notes_pos = datadoc.find('Notes\n    -----')
+            existing_notes_pos2 = notes_doc.find('Notes\n-----')
+            existing_notes_data = textwrap.indent(notes_doc[existing_notes_pos2 + len('Notes\n-----'):].strip(), "    ")
+            references_pattern = "References\n    ----------"
+            examples_pattern = "Examples\n   -------"
+            start_index = existing_notes_pos + len('Notes\n    -----')
+            references_pos = datadoc.find(references_pattern, start_index)
+            examples_pos = datadoc.find(examples_pattern, start_index)
+            if existing_notes_pos != -1:
```

Comment:
Can this set of if/else blocks not be simplified?

Reply:
I tried doing something, let me know if its okay now!